### PR TITLE
feat: add GPU tab to display VRAM info

### PR DESCRIPTION
**Pull request by ajite**
Hi,

I thought it will be nice to have a tab to display the VRAM and teach people how to change it. (native speaker might help to review my english)

I am not proficient in go and tried to adapt to your style. I did not see any Unit Tests but I could add some. (e.g.: for the regex parsing).

### CODE REVIEW

**Comment by CryoByte33** [comment]
This is a great idea! 

That said, the button feels misleading, like it will help you change it right in the UI. Maybe it should just be instructions directly in the tab instead, like a well-formatted text box? 

I would also say that we should have the current setting "human readable" for everyone out there who doesn't know base-2 math 😉

**Comment by ajite** [comment]
> the button feels misleading

I agree. I felt that using the information window made it look cleaner than something like below:

Screenshot_20230223_130057

We could also change the button text to: `Show how to increase VRAM`

What do you think ?

**Comment by CryoByte33** [comment]
> We could also change the button text to: Show how to increase VRAM
I think `GPU` is a little off, but I vote for `VRAM` to keep it short.

Also, would you mind changing the wording in the window to be like this? It should remove a few grammatical issues and clarify:

----------

# Minimum VRAM
## How to change the minimum VRAM:
1. Turn off the Steam Deck
2. Press and hold the volume up button, press the power button, then release both
3. Navigate to Setup Utility > Advanced > UMA Frame Buffer Size

**4G is the recommended setting for most situations.**

Please be aware that some games (RDR2) may experience degraded performance.

**Comment by ajite** [comment]
Great. It is much better like that.

Do you prefer option A or B.

### Option A 



### Option B

**Comment by CryoByte33** [comment]
I prefer option A, it definitely seems cleaner.

Thank you for all the work you're putting into it!

**Comment by ajite** [comment]
> Thank you for all the work you're putting into it!

Thanks to you. My games run much better on my Steam Deck. I always wanted to learn how to develop in Go. That's a good opportunity.

I uploaded my change. I will try them on my Deck later today.

File: internal/util.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ type Config struct {
 	MemoryContainer               *fyne.Container
 	SwapBar                       *fyne.Container
 	MemoryBar                     *fyne.Container
-	GPUBar                        *fyne.Container
+	VRAMBar                        *fyne.Container
```
Comment (CryoByte33) [inline internal/util.go:43]: The VRAM bar shouldn't need to be added to the app object, since it doesn't need to refresh/have a receiver.

Comment (ajite) [reply internal/util.go:43]: make sense. I ll update and squash my commits into 1.

**Comment by CryoByte33** [comment]
It looks good other than the one comment, which I see you're addressing. Also having unit tests is a major win, I really need to add them 😅

Pending testing on Deck, this looks basically good to go!

**Comment by ajite** [comment]
It works on my Deck. 

I tried with both 4GB and 256MB

**Comment by CryoByte33** [comment]
This might be a little bit of a nitpick, but I was finally able to test. Could you swap `theme.MediaVideoIcon()` with `theme.ViewFullScreenIcon()` for the tab icon? The camera is a bit off to me and I like the feel of the other better.

**Comment by CryoByte33** [comment]
I've decided to just fix this post-merge since it's not "really" an issue.

**Pull request merged**